modules/op-about-tasks.adoc

```diff
@@ -36,9 +36,27 @@ spec: <4>
           oc apply -f $(params.manifest_dir)
           echo -----------------------------------
 ----
-<1> The Task API version, `v1beta1`.
+<1> The task API version, `v1beta1`.
 <2> The type of Kubernetes object, `Task`.
-<3> The unique name of this Task.
-<4> The list of parameters and Steps in the Task and the workspace used by the Task.
+<3> The unique name of this task.
+<4> The list of parameters and steps in the task and the workspace used by the task.
 
-This Task starts the pod and runs a container inside that pod using the specified image to run the specified commands.
+This task starts the pod and runs a container inside that pod using the specified image to run the specified commands.
+
+[NOTE]
+====
+Starting with {pipelines-shortname} {pipelines-ver}, the steps in a task does not have the following defaults:
+
+* The `HOME` environment variable set to `/tekton/home`
+* The `workingDir` field set to `/workspace`
+
+Instead, the step's container defines the `HOME` environment variable and the `workingDir` field. However, you can override the default values by specifying the custom values in the step's YAML.
```

Comment:
the step's container/the container for the step - Avoid possessives for inanimate objects
step's YAML/step YAML file or the YAML file for the step.